app/controllers/api/v1/facts_controller.rb

```diff
@@ -0,0 +1,62 @@
+class Api::V1::FactsController < ApplicationController
+    include AuthenticationCheck
+  
+    before_action :is_user_logged_in
+    before_action :set_fact, only: [:show, :update, :destroy]
+  
+    # GET /members/:member_id/facts
+    def index
+      @member = Member.find(params[:member_id])
+      render json: @member.facts # note that because the facts route is nested inside members
+                               # we return only the facts belonging to that member
+    end
+  
+    # GET /members/:member_id/facts/:id
+    def show
+      # your code goes here
+      render json: @fact
+    end
+  
+    # POST /members/:member_id/facts
+    def create
+       @member = Member.find(params[:member_id])
+      @fact = @member.facts.new(fact_params)
+      if @fact.save
+        render json: @fact, status: 201
+      else
+        render json: { error: 
+  "The fact entry could not be created. #{@fact.errors.full_messages.to_sentence}"},
+        status: 400
+      end
+    end
+  
+    # PUT /members/:member_id/facts/:id
+    def update
+      # your code goes here
+      if @fact.update(fact_params)
+        render json: @fact, status: 201
```

Comment:
HTTP status 201 means "Created", but that's not what an update does, so 200 would be more appropriate. As a side note, having to remember what every code does in order to read source code is not great, which is why Rails allows for symbols to be used instead (full list [here](http://www.railsstatuscodes.com/)), and that's what we usually use.

```suggestion
        render json: @fact, status: :ok
```